contrib/android/Dockerfile

```diff
@@ -179,9 +195,10 @@ RUN cd /opt \
     && git remote add sombernight https://github.com/SomberNight/python-for-android \
     && git remote add accumulator https://github.com/accumulator/python-for-android \
     && git fetch --all \
-    # commit: from branch accumulator/electrum_20210421d (note: careful with force-pushing! see #8162)
-    && git checkout "052b9f7945bae557347fa4a4b418040d9da9eaff^{commit}" \
-    && python3 -m pip install --no-build-isolation --no-dependencies --user -e .
+    # commit: from branch accumulator/qt6-wip (note: careful with force-pushing! see #8162) \
+    #
+    && git checkout "46afc71fc0ba501f351c83cfcaac5504c7c935d6^{commit}" \
```

Comment:
> APK versionCode is still a TODO. See p4a commits:
>    [f5440f78ce5f1f2492aa87d845ccc163067aeda4](https://github.com/SomberNight/python-for-android/commit/f5440f78ce5f1f2492aa87d845ccc163067aeda4)
>    [72782ed93de7515965388aff66222aa34dbf7ee4](https://github.com/accumulator/python-for-android/commit/72782ed93de7515965388aff66222aa34dbf7ee4)

see also current commit used for already released versions:
https://github.com/accumulator/python-for-android/commit/3a52f805bec49a6818d527286ce5fd09823f738a
so e.g. 4.4.5(.0) for x86_64, I expect has versionCode `4_4_04_05_00 == 44040500`.

1. the versionCode needs to strictly increase between releases
2. there cannot be a collision of versionCodes between different apks
    - as we don't build multi-arch apks, this means versionCodes need to differ between archs 
3. do we want to build apks with different minSDKs? if not, we can leave that out of the versionCode
4. note that the trailing `.0` in the `APK_VERSION` in `electrum/version.py` has been unused for a long time -- it was only used in the pre-reproducible-builds era. so potentially that could be removed
5. as in https://github.com/kivy/python-for-android/issues/1843, the address space is only 32-bit (not sure if signed or unsigned)
    - any "jump" we make, burns address space we never get back

Disregarding (4), I think [f5440f78ce5f1f2492aa87d845ccc163067aeda4](https://github.com/SomberNight/python-for-android/commit/f5440f78ce5f1f2492aa87d845ccc163067aeda4) is the easiest approach.

Reply:
oh wait, [f5440f78ce5f1f2492aa87d845ccc163067aeda4](https://github.com/SomberNight/python-for-android/commit/f5440f78ce5f1f2492aa87d845ccc163067aeda4) does not deal with (2) either -- unless we will start building multi-arch apks now

Reply:
I restored the behavior of https://github.com/accumulator/python-for-android/commit/3a52f805bec49a6818d527286ce5fd09823f738a, so it is as in current releases.